### PR TITLE
Adaptive Explore window and 100% height

### PR DESCRIPTION
**Pull request by JeanPaulLucien**
- increased height;
- decreased width;
- changed minimum window's sizes for small screens;
- changed room avatar size and increased padding to not break;

More rooms are visible by default. I really would to make 100% height.

### CODE REVIEW

File: resources/qml/dialogs/RoomDirectory.qml [hunk 1]
```diff
@@ -16,10 +16,10 @@ ApplicationWindow {
     id: roomDirectoryWindow
 
     visible: true
-    minimumWidth: 340
-    minimumHeight: 340
-    height: 420
-    width: 650
+    minimumHeight 320 
+    minimumWidth: 460
```
Comment (deepbluev7) [inline resources/qml/dialogs/RoomDirectory.qml:20]: This makes the window too big for some phone screens, afaik.

**Comment by deepbluev7** [comment]
Hm, why do you want the window to be a different size?

File: resources/qml/dialogs/RoomDirectory.qml (hunk 1 above)
Comment (JeanPaulLucien) [reply resources/qml/dialogs/RoomDirectory.qml:20]: Okay for 720*1280. Even for 480p.

**Comment by JeanPaulLucien** [comment]
@deepbluev7 
Yeah, I'm agree. 460 was many. It's better now.

File: resources/qml/dialogs/RoomDirectory.qml (hunk 1 above)
Comment (deepbluev7) [reply resources/qml/dialogs/RoomDirectory.qml]: The PinePhone uses a rotated 720p screen with 2x display scaling or so. That tends to cause issues :3

**Comment by deepbluev7** [comment]
I don't think I want the window narrower. It makes the topics harder to read.

**Comment by JeanPaulLucien** [comment]
> I don't think I want the window narrower. It makes the topics harder to read.

It's for mobile phones + you can use build-in desktop OS feature to make it half or 1/4 of screen.

**Comment by JeanPaulLucien** [comment]
@deepbluev7 
Review and merge please if it's okay.

Changes:
- height is 100% now;
- it's adaptive window;
I hope it will work well for each screens.

**Comment by deepbluev7** [comment]
This doesn't work. Using the full screen size breaks on multiple monitors.

**Pull request closed without merging**